---
layout: default
title: "Inputs: PairedCnvCalling"
---

# Inputs for PairedCnvCalling

The following is an overview of all available inputs in
PairedCnvCalling.


## Required inputs
<dl>
<dt id="PairedCnvCalling.caseBam"><a href="#PairedCnvCalling.caseBam">PairedCnvCalling.caseBam</a></dt>
<dd>
    <i>File </i><br />
    The BAM file for the case sample.
</dd>
<dt id="PairedCnvCalling.caseBamIndex"><a href="#PairedCnvCalling.caseBamIndex">PairedCnvCalling.caseBamIndex</a></dt>
<dd>
    <i>File </i><br />
    The index for the case sample's BAM file.
</dd>
<dt id="PairedCnvCalling.caseSampleName"><a href="#PairedCnvCalling.caseSampleName">PairedCnvCalling.caseSampleName</a></dt>
<dd>
    <i>String </i><br />
    The name of the case sample.
</dd>
<dt id="PairedCnvCalling.commonVariantSites"><a href="#PairedCnvCalling.commonVariantSites">PairedCnvCalling.commonVariantSites</a></dt>
<dd>
    <i>File </i><br />
    Interval list or VCF file of common variant sites (to retrieve the allelic counts for). Preferably a list variants from the sample being analysed. For targeted/exome sequencing the list should be limited to variants within the sequenced regions, due to memory usage.
</dd>
<dt id="PairedCnvCalling.controlBam"><a href="#PairedCnvCalling.controlBam">PairedCnvCalling.controlBam</a></dt>
<dd>
    <i>File </i><br />
    The BAM file for the control sample.
</dd>
<dt id="PairedCnvCalling.controlBamIndex"><a href="#PairedCnvCalling.controlBamIndex">PairedCnvCalling.controlBamIndex</a></dt>
<dd>
    <i>File </i><br />
    The index for the control sample's BAM file.
</dd>
<dt id="PairedCnvCalling.controlSampleName"><a href="#PairedCnvCalling.controlSampleName">PairedCnvCalling.controlSampleName</a></dt>
<dd>
    <i>String </i><br />
    The name of the control sample.
</dd>
<dt id="PairedCnvCalling.preprocessedIntervals"><a href="#PairedCnvCalling.preprocessedIntervals">PairedCnvCalling.preprocessedIntervals</a></dt>
<dd>
    <i>File </i><br />
    Intervals to operate on. Should be produced by gatk PreprocessIntervals (eg. using CNV-PON.wdl).
</dd>
<dt id="PairedCnvCalling.referenceFasta"><a href="#PairedCnvCalling.referenceFasta">PairedCnvCalling.referenceFasta</a></dt>
<dd>
    <i>File </i><br />
    The reference fasta file.
</dd>
<dt id="PairedCnvCalling.referenceFastaDict"><a href="#PairedCnvCalling.referenceFastaDict">PairedCnvCalling.referenceFastaDict</a></dt>
<dd>
    <i>File </i><br />
    The sequence dictionary associated with the reference fasta file.
</dd>
<dt id="PairedCnvCalling.referenceFastaFai"><a href="#PairedCnvCalling.referenceFastaFai">PairedCnvCalling.referenceFastaFai</a></dt>
<dd>
    <i>File </i><br />
    The index for the reference fasta file.
</dd>
</dl>

## Other common inputs
<dl>
<dt id="PairedCnvCalling.annotatedIntervals"><a href="#PairedCnvCalling.annotatedIntervals">PairedCnvCalling.annotatedIntervals</a></dt>
<dd>
    <i>File? </i><br />
    An annotated set of intervals as generated by AnnotateIntervals (eg. using CNV-PON.wdl). Will be ignored if PON is provided.
</dd>
<dt id="PairedCnvCalling.commonVariantSitesIndex"><a href="#PairedCnvCalling.commonVariantSitesIndex">PairedCnvCalling.commonVariantSitesIndex</a></dt>
<dd>
    <i>File? </i><br />
    The index for commonVariantSitesIndex.
</dd>
<dt id="PairedCnvCalling.outputDir"><a href="#PairedCnvCalling.outputDir">PairedCnvCalling.outputDir</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"."</code><br />
    The directory the output should be written to.
</dd>
<dt id="PairedCnvCalling.PON"><a href="#PairedCnvCalling.PON">PairedCnvCalling.PON</a></dt>
<dd>
    <i>File? </i><br />
    A read counts panel of normals as generated by gatk CreateReadCountPanelOfNormals (eg. using CNV-PON.wdl).
</dd>
</dl>

## Advanced inputs
<details>
<summary> Show/Hide </summary>
<dl>
<dt id="PairedCnvCalling.caseSample.callCopyRatioSegments.javaXmx"><a href="#PairedCnvCalling.caseSample.callCopyRatioSegments.javaXmx">PairedCnvCalling.caseSample.callCopyRatioSegments.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"2G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.caseSample.callCopyRatioSegments.memory"><a href="#PairedCnvCalling.caseSample.callCopyRatioSegments.memory">PairedCnvCalling.caseSample.callCopyRatioSegments.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"3G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.caseSample.callCopyRatioSegments.timeMinutes"><a href="#PairedCnvCalling.caseSample.callCopyRatioSegments.timeMinutes">PairedCnvCalling.caseSample.callCopyRatioSegments.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>2</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.caseSample.collectAllelicCounts.javaXmx"><a href="#PairedCnvCalling.caseSample.collectAllelicCounts.javaXmx">PairedCnvCalling.caseSample.collectAllelicCounts.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"10G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.caseSample.collectAllelicCounts.memory"><a href="#PairedCnvCalling.caseSample.collectAllelicCounts.memory">PairedCnvCalling.caseSample.collectAllelicCounts.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"11G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.caseSample.collectAllelicCounts.timeMinutes"><a href="#PairedCnvCalling.caseSample.collectAllelicCounts.timeMinutes">PairedCnvCalling.caseSample.collectAllelicCounts.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>120</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.caseSample.collectReadCounts.intervalMergingRule"><a href="#PairedCnvCalling.caseSample.collectReadCounts.intervalMergingRule">PairedCnvCalling.caseSample.collectReadCounts.intervalMergingRule</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"OVERLAPPING_ONLY"</code><br />
    Equivalent to gatk CollectReadCounts' `--interval-merging-rule` option.
</dd>
<dt id="PairedCnvCalling.caseSample.collectReadCounts.javaXmx"><a href="#PairedCnvCalling.caseSample.collectReadCounts.javaXmx">PairedCnvCalling.caseSample.collectReadCounts.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"7G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.caseSample.collectReadCounts.memory"><a href="#PairedCnvCalling.caseSample.collectReadCounts.memory">PairedCnvCalling.caseSample.collectReadCounts.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"8G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.caseSample.collectReadCounts.timeMinutes"><a href="#PairedCnvCalling.caseSample.collectReadCounts.timeMinutes">PairedCnvCalling.caseSample.collectReadCounts.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>1 + ceil((size(inputBam,"G") * 5))</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.caseSample.denoiseReadCounts.javaXmx"><a href="#PairedCnvCalling.caseSample.denoiseReadCounts.javaXmx">PairedCnvCalling.caseSample.denoiseReadCounts.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"4G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.caseSample.denoiseReadCounts.memory"><a href="#PairedCnvCalling.caseSample.denoiseReadCounts.memory">PairedCnvCalling.caseSample.denoiseReadCounts.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"5G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.caseSample.denoiseReadCounts.timeMinutes"><a href="#PairedCnvCalling.caseSample.denoiseReadCounts.timeMinutes">PairedCnvCalling.caseSample.denoiseReadCounts.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>5</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.caseSample.modelSegments.javaXmx"><a href="#PairedCnvCalling.caseSample.modelSegments.javaXmx">PairedCnvCalling.caseSample.modelSegments.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"10G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.caseSample.modelSegments.maximumNumberOfSmoothingIterations"><a href="#PairedCnvCalling.caseSample.modelSegments.maximumNumberOfSmoothingIterations">PairedCnvCalling.caseSample.modelSegments.maximumNumberOfSmoothingIterations</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>10</code><br />
    Equivalent to gatk ModelSeqments' `--maximum-number-of-smoothing-iterations` option.
</dd>
<dt id="PairedCnvCalling.caseSample.modelSegments.memory"><a href="#PairedCnvCalling.caseSample.modelSegments.memory">PairedCnvCalling.caseSample.modelSegments.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"11G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.caseSample.modelSegments.minimumTotalAlleleCountCase"><a href="#PairedCnvCalling.caseSample.modelSegments.minimumTotalAlleleCountCase">PairedCnvCalling.caseSample.modelSegments.minimumTotalAlleleCountCase</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>if defined(normalAllelicCounts) then 0 else 30</code><br />
    Equivalent to gatk ModelSeqments' `--minimum-total-allele-count-case` option.
</dd>
<dt id="PairedCnvCalling.caseSample.modelSegments.timeMinutes"><a href="#PairedCnvCalling.caseSample.modelSegments.timeMinutes">PairedCnvCalling.caseSample.modelSegments.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>60</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.caseSample.plotDenoisedCopyRatios.javaXmx"><a href="#PairedCnvCalling.caseSample.plotDenoisedCopyRatios.javaXmx">PairedCnvCalling.caseSample.plotDenoisedCopyRatios.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"3G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.caseSample.plotDenoisedCopyRatios.memory"><a href="#PairedCnvCalling.caseSample.plotDenoisedCopyRatios.memory">PairedCnvCalling.caseSample.plotDenoisedCopyRatios.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"4G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.caseSample.plotDenoisedCopyRatios.timeMinutes"><a href="#PairedCnvCalling.caseSample.plotDenoisedCopyRatios.timeMinutes">PairedCnvCalling.caseSample.plotDenoisedCopyRatios.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>2</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.caseSample.plotModeledSegments.javaXmx"><a href="#PairedCnvCalling.caseSample.plotModeledSegments.javaXmx">PairedCnvCalling.caseSample.plotModeledSegments.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"3G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.caseSample.plotModeledSegments.memory"><a href="#PairedCnvCalling.caseSample.plotModeledSegments.memory">PairedCnvCalling.caseSample.plotModeledSegments.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"4G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.caseSample.plotModeledSegments.timeMinutes"><a href="#PairedCnvCalling.caseSample.plotModeledSegments.timeMinutes">PairedCnvCalling.caseSample.plotModeledSegments.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>2</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.controlSample.callCopyRatioSegments.javaXmx"><a href="#PairedCnvCalling.controlSample.callCopyRatioSegments.javaXmx">PairedCnvCalling.controlSample.callCopyRatioSegments.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"2G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.controlSample.callCopyRatioSegments.memory"><a href="#PairedCnvCalling.controlSample.callCopyRatioSegments.memory">PairedCnvCalling.controlSample.callCopyRatioSegments.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"3G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.controlSample.callCopyRatioSegments.timeMinutes"><a href="#PairedCnvCalling.controlSample.callCopyRatioSegments.timeMinutes">PairedCnvCalling.controlSample.callCopyRatioSegments.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>2</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.controlSample.collectAllelicCounts.javaXmx"><a href="#PairedCnvCalling.controlSample.collectAllelicCounts.javaXmx">PairedCnvCalling.controlSample.collectAllelicCounts.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"10G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.controlSample.collectAllelicCounts.memory"><a href="#PairedCnvCalling.controlSample.collectAllelicCounts.memory">PairedCnvCalling.controlSample.collectAllelicCounts.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"11G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.controlSample.collectAllelicCounts.timeMinutes"><a href="#PairedCnvCalling.controlSample.collectAllelicCounts.timeMinutes">PairedCnvCalling.controlSample.collectAllelicCounts.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>120</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.controlSample.collectReadCounts.intervalMergingRule"><a href="#PairedCnvCalling.controlSample.collectReadCounts.intervalMergingRule">PairedCnvCalling.controlSample.collectReadCounts.intervalMergingRule</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"OVERLAPPING_ONLY"</code><br />
    Equivalent to gatk CollectReadCounts' `--interval-merging-rule` option.
</dd>
<dt id="PairedCnvCalling.controlSample.collectReadCounts.javaXmx"><a href="#PairedCnvCalling.controlSample.collectReadCounts.javaXmx">PairedCnvCalling.controlSample.collectReadCounts.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"7G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.controlSample.collectReadCounts.memory"><a href="#PairedCnvCalling.controlSample.collectReadCounts.memory">PairedCnvCalling.controlSample.collectReadCounts.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"8G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.controlSample.collectReadCounts.timeMinutes"><a href="#PairedCnvCalling.controlSample.collectReadCounts.timeMinutes">PairedCnvCalling.controlSample.collectReadCounts.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>1 + ceil((size(inputBam,"G") * 5))</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.controlSample.denoiseReadCounts.javaXmx"><a href="#PairedCnvCalling.controlSample.denoiseReadCounts.javaXmx">PairedCnvCalling.controlSample.denoiseReadCounts.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"4G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.controlSample.denoiseReadCounts.memory"><a href="#PairedCnvCalling.controlSample.denoiseReadCounts.memory">PairedCnvCalling.controlSample.denoiseReadCounts.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"5G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.controlSample.denoiseReadCounts.timeMinutes"><a href="#PairedCnvCalling.controlSample.denoiseReadCounts.timeMinutes">PairedCnvCalling.controlSample.denoiseReadCounts.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>5</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.controlSample.modelSegments.javaXmx"><a href="#PairedCnvCalling.controlSample.modelSegments.javaXmx">PairedCnvCalling.controlSample.modelSegments.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"10G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.controlSample.modelSegments.maximumNumberOfSmoothingIterations"><a href="#PairedCnvCalling.controlSample.modelSegments.maximumNumberOfSmoothingIterations">PairedCnvCalling.controlSample.modelSegments.maximumNumberOfSmoothingIterations</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>10</code><br />
    Equivalent to gatk ModelSeqments' `--maximum-number-of-smoothing-iterations` option.
</dd>
<dt id="PairedCnvCalling.controlSample.modelSegments.memory"><a href="#PairedCnvCalling.controlSample.modelSegments.memory">PairedCnvCalling.controlSample.modelSegments.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"11G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.controlSample.modelSegments.minimumTotalAlleleCountCase"><a href="#PairedCnvCalling.controlSample.modelSegments.minimumTotalAlleleCountCase">PairedCnvCalling.controlSample.modelSegments.minimumTotalAlleleCountCase</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>if defined(normalAllelicCounts) then 0 else 30</code><br />
    Equivalent to gatk ModelSeqments' `--minimum-total-allele-count-case` option.
</dd>
<dt id="PairedCnvCalling.controlSample.modelSegments.timeMinutes"><a href="#PairedCnvCalling.controlSample.modelSegments.timeMinutes">PairedCnvCalling.controlSample.modelSegments.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>60</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.controlSample.plotDenoisedCopyRatios.javaXmx"><a href="#PairedCnvCalling.controlSample.plotDenoisedCopyRatios.javaXmx">PairedCnvCalling.controlSample.plotDenoisedCopyRatios.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"3G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.controlSample.plotDenoisedCopyRatios.memory"><a href="#PairedCnvCalling.controlSample.plotDenoisedCopyRatios.memory">PairedCnvCalling.controlSample.plotDenoisedCopyRatios.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"4G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.controlSample.plotDenoisedCopyRatios.timeMinutes"><a href="#PairedCnvCalling.controlSample.plotDenoisedCopyRatios.timeMinutes">PairedCnvCalling.controlSample.plotDenoisedCopyRatios.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>2</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.controlSample.plotModeledSegments.javaXmx"><a href="#PairedCnvCalling.controlSample.plotModeledSegments.javaXmx">PairedCnvCalling.controlSample.plotModeledSegments.javaXmx</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"3G"</code><br />
    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
</dd>
<dt id="PairedCnvCalling.controlSample.plotModeledSegments.memory"><a href="#PairedCnvCalling.controlSample.plotModeledSegments.memory">PairedCnvCalling.controlSample.plotModeledSegments.memory</a></dt>
<dd>
    <i>String </i><i>&mdash; Default:</i> <code>"4G"</code><br />
    The amount of memory this job will use.
</dd>
<dt id="PairedCnvCalling.controlSample.plotModeledSegments.timeMinutes"><a href="#PairedCnvCalling.controlSample.plotModeledSegments.timeMinutes">PairedCnvCalling.controlSample.plotModeledSegments.timeMinutes</a></dt>
<dd>
    <i>Int </i><i>&mdash; Default:</i> <code>2</code><br />
    The maximum amount of time the job will run in minutes.
</dd>
<dt id="PairedCnvCalling.dockerImages"><a href="#PairedCnvCalling.dockerImages">PairedCnvCalling.dockerImages</a></dt>
<dd>
    <i>Map[String,String] </i><i>&mdash; Default:</i> <code>{"gatk": "broadinstitute/gatk:4.1.8.0"}</code><br />
    The docker images used. Changing this may result in errors which the developers may choose not to address.
</dd>
<dt id="PairedCnvCalling.minimumContigLength"><a href="#PairedCnvCalling.minimumContigLength">PairedCnvCalling.minimumContigLength</a></dt>
<dd>
    <i>Int? </i><br />
    The minimum length for a contig to be included in the plots.
</dd>
</dl>
</details>





## Do not set these inputs!
The following inputs should ***not*** be set, even though womtool may
show them as being available inputs.

* PairedCnvCalling.controlSample.matchedNormalAllelicCounts
